robotd/visiond/tokens.py

```diff
@@ -0,0 +1,74 @@
+import code
+from visiond.apriltag._apriltag import ffi
+import numpy as np
+import math
+
+
+def get_pixel_corners(homog):
+    # Define the corners of the marker
+    corners = np.array([(-1, -1), (-1, 1), (1, 1), (1, -1)])
+
+    # Perform the equivalent of an OpenCV WarpPerspectiveTransform on the points
+    # See http://docs.opencv.org/2.4/modules/imgproc/doc/geometric_transformations.html#initundistortrectifymap
+    # for the equation
+
+    def row_mul(m, corner, col):
+        return m[col, 0] * corner[0] + m[col, 1] * corner[1] + m[col, 2]
```

Comment:
This looks like a straightforward matrix multiplication—do we need to write this out longhand?